### PR TITLE
verbiage for new workshop landing page

### PR DESCRIPTION
**Pull request by eitansuez**
adds a workshop markdown page for upcoming workshop on cnd developer practices.

### CODE REVIEW

File: content/workshops/cnd-deploy-practices.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+draft: false
+title: "Cloud Native Developer practices on Kubernetes"
```
Comment (tybritten) [inline content/workshops/cnd-deploy-practices.md:3]:
```suggestion
title: "Cloud Native Developer Practices on Kubernetes"
```

File: content/workshops/cnd-deploy-practices.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+In this workshop you will:
+
+- Review and run a basic Spring Boot application
+- Build a Docker image for the application
+- Push the image to a Docker registry
+- Deploy and run the application on Kubernetes
+- Configure and test an ingress for the application
+
+This workshop is intended to become the first in a series of workshops that explore cloud-native developer practices.
```
Comment (tybritten) [inline content/workshops/cnd-deploy-practices.md:32]:
```suggestion
This workshop is the first in a series of workshops that explore cloud-native developer practices.
```

**Review by tybritten — CHANGES_REQUESTED** [review]

File: content/workshops/cnd-deploy-practices.md (hunk 1 above)
Comment (eitansuez) [reply content/workshops/cnd-deploy-practices.md:3]: thanks for the review.  changes look good to me. :-)

**Comment by billkable** [comment]
@tybritten Just an FYI I will be submitting a new PR on this with different content / objectives.

We re-vectored a bit on the workshop objectives and outcomes to be less Spring/K8s and more practices and patterns coverage.

-B

**Comment by tybritten** [comment]
thanks for the heads up. you can just use this PR and push a change

**Comment by bryanfriedman** [comment]
@tybritten Do we still need this PR or should we close?

**Comment by tybritten** [comment]
Nope, closed

**Pull request closed without merging**